### DISCUSSION TITLE
Forking JS keycloak has issue

### DISCUSSION THREAD

**Discussion by thaitv17** [keycloak/keycloak#29874] [Ideas]
Hi everyone, i am a newbie to keycloak and now i have some tasks to migrate to keycloak 21. Because i have custom extension so it need to migrate from angular to react. I have read on discussion that team keycloak they suggested we should fork the repo and then build our custom. My issue right now is that if i copy js folder to another project (my own project), i can not run admin-ui locally anymore.
Current behavior: 

1. when i run in admin-ui or keycloak-server, i can not run pnpm anymore
```pnpm dev                  


> admin-ui@ dev /home/thaito/Desktop/keycloak-up-extensions/js/apps/admin-ui
> wireit

sh: 1: wireit: not found
 ELIFECYCLE  Command failed.
```

Could anyone can help me! Thanks for your support.

**Comment by Sieboldianus** [comment]
This can be fixed with:
```bash
pnpm i -D wireit
```

**Reply by thaitv17** [reply]
Thank you!
